content/python/concepts/strings/terms/isalpha/isalpha.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: '.isalpha()'
+Description: 'Returns True if all characters in a string are alphabet letters (a-z), otherwise it returns False.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.isalpha()`** string method checks if all of the characters in a string are alphabet letters (a-z). The letters can be lowercase or uppercase. If the string contains only alphabet letters it returns `True`, otherwise it returns `False`. Whitespace, numbers, and symbols are not considered to be alphabet characters.
```

Comment:
```suggestion
The **`.isalpha()`** string method checks if all of the characters in a string are letters of the alphabet (`a-z`). The letters can be lowercase or uppercase. If the string contains only alphabet letters it returns `True`, otherwise it returns `False`. Whitespace, numbers, and symbols are not considered to be part of the alphabet.
```